interface/main/tabs/js/tabs_view_model.js

```diff
@@ -19,15 +19,20 @@ function tabStatus(title,url,name,closable,visible,locked)
     self.title=ko.observable(title);
     self.url=ko.observable(url);
     self.name=ko.observable(name);
+    self.loading_text=ko.observable("Loading " + label + "...");
+    self.loader = ko.observable(true);
+    self.title.subscribe(function() {
+        self.loader(false);
+    });
     self.window=null;
     return this;
 }
 
 function tabs_view_model()
 {
     this.tabsList=ko.observableArray();
-    this.tabsList.push(new tabStatus("Loading...",webroot_url+"/interface/main/main_info.php","cal",true,true,false));
-    this.tabsList.push(new tabStatus("Loading...",webroot_url+"/interface/main/messages/messages.php?form_active=1","msg",true,false,false));
+    this.tabsList.push(new tabStatus("Loading <a style='font-size:80%'><i class='fa fa-spinner fa-pulse'></i></a>",webroot_url+"/interface/main/main_info.php","cal","Calender",true,true,false));
+    this.tabsList.push(new tabStatus("Loading <a style='font-size:80%'><i class='fa fa-spinner fa-pulse'></i></a>",webroot_url+"/interface/main/messages/messages.php?form_active=1","Messages","msg",true,false,false));
```

Comment:
Also, lets translate the Loading label, by placing this in place of Loading:
```
<?php echo xla("Loading"); ?>
```

Reply:
oh, wait, this is js, so can't use php. lets use the javascript translate function then instead:
```
xl("loading") + "<a style...
```

Reply:
@bradymiller  here it gives undefined 😕

Reply:
further debugging 
![image](https://user-images.githubusercontent.com/47683556/76171105-ce80b780-61ad-11ea-9da1-ac6cf5e0f890.png)